get_youtube_view.py

```diff
@@ -1,10 +1,23 @@
+"""
+Created on Thu Apr 27 16:28:36 2017
+@author: barnabysandeford
+"""
+# Currently works for Safari, but just change to whichever 
+# browser you're using
+
```

Comment:
Please add comment explaining what the code is achieving. I can see that part in deleted. It would be helpful to understand.